docs/advanced-topics/transports.md

```diff
@@ -22,6 +22,94 @@ A transport layer can provide:
+
+### Setting up a single player session
+In addition to your default network transport, you will want to add the `SinglePlayerTransport` to the `NetworkManager` `GameObject` (or child of). 
+![image](/img/transport/SinglePlayerTransport_AddComponent.png)
+
+In order to start a single player session you must make sure to assign the `SinglePlayerTransport` to the `NetworkManager.NetworkConfig.NetworkTransport` configuration property using script. 
+As an example:
+
+```csharp
+using Unity.Netcode;
+using Unity.Netcode.Transports.UTP;
+using Unity.Netcode.Transports.SinglePlayer;
+using UnityEngine;
+
+public class ExtendedNetworkManager : NetworkManager
+{
+    private UnityTransport m_UnityTransport;
+    private SinglePlayerTransport m_SinglePlayerTransport;
+
+    private void Awake()
+    {
+        m_UnityTransport = GetComponent<UnityTransport>();
+        m_SinglePlayerTransport = GetComponent<SinglePlayerTransport>();
+    }
+
+    public void StartSinglePlayer()
+    {
+        // Use the single player transport when starting a single player session.
+        NetworkConfig.NetworkTransport = m_SinglePlayerTransport;
+        if (!StartHost())
+        {
+            NetworkLog.LogError("Failed to start single player session!");
+        }
+    }
+
+    public void StartHostedSession()
+    {
+        // Use the network transport when starting a multiplayer session.
+        NetworkConfig.NetworkTransport = m_UnityTransport;
+        if (!StartHost())
+        {
+            NetworkLog.LogError("Failed to start hosted session!");
+        }
+    }
+}
+```
+
+From the above script, when starting a single player session the `SinglePlayerTransport` is assigned to the `NetworkConfig.NetworkTransport` and when starting a hosted multiplayer session the `UnityTransport` is assigned. 
+
+### Single player limitations
+
+When running a single player session, you should take any netcode script into consideration that requires an actual multiplayer session to exist.
+
+#### RPC considerations
+You can invoke any RPC that includes:
+
+- The host's client.
+- The host's server.
+- `SendTo` targets that will be invoked locally:
+  - `SendTo.Me`
+  - `SendTo.Server`
+  - `SendTo.Everyone`
+  - `SendTo.Authority`
+  - `SendTo.Owner`
+  - `ClientsAndHost`
+  - `SpecifiedInParams`: As long as it is targeting the `NetworkManager.LocalClientId`.
+- `SendTo` targets that will **not be** invoked locally:
+  - `SendTo.NotOwner`
+  - `SendTo.NotServer`
+  - `SendTo.NotMe`
+  - `SendTo.NotAuthority`
+  - `SpecifiedInParams`: Anything not targeting the `NetworkManager.LocalClientId` will not be invoked locally.
+
+#### NetworkVariable considerations
+
+This should work as expected as the host will always be both the server and the owner of anything spawned. This means write permissions, whether server or owner, should not be an issue.
```

Comment:
```suggestion
NetworkVariables should work as expected because the host will always be both the server and the owner of anything spawned. This means that write permissions, whether server or owner, should not be an issue.
```